bitrise.yml

```diff
@@ -2272,6 +2272,150 @@ workflows:
             focus-ios/focus-ios-tests/tools/sentry-cli --auth-token "$SENTRY_AUTH_TOKEN" upload-dif \
               --org mozilla --project klar-ios "$BITRISE_DSYM_DIR_PATH"
 
+# RelPro workflows
+  release_promotion_promote:
+    before_run:
+    - SPM_Common_Steps
+    steps:
+    - script@1.2.1:
+        title: Set staging environment variables
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -ex
+            envman add --key BITRISE_SCHEME --value FirefoxStaging
+            envman add --key EXPORT_METHOD --value development
+        run_if: '{{envcontain "GIT_REPOSITORY_URL" "staging"}}'
+    - script@1.2.1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -e
+            set -x
+
+            cd firefox-ios/Client.xcodeproj
+            sed -i '' 's/CODE_SIGN_IDENTITY = "iPhone Developer"/CODE_SIGN_IDENTITY = "iPhone Distribution"/' project.pbxproj
+            cd -
+        title: Set xcodeproj code_sign_identity
+        run_if: '{{getenv "BITRISE_SCHEME" | ne "FirefoxStaging"}}'
```

Comment:
Same thing here, maybe change this to `{{enveq "BITRISE_APP_SLUG" "firefox-ios"}}`?